examples/test/integrations.test.js

```diff
@@ -0,0 +1,37 @@
+const test = require("ava");
+const { server, app } = require("../integrations/server");
+const port = 4000;
+app.listen({ port });
+const sendQuery = query =>
+  require("./util/sendQuery")(query, server.graphqlPath, port);
+
+const query = `
+query getCharacterInfo {
+  getCharacterInfo(id:11) {
+    name 
+    surname
+    height
+    gender
+    genderFromName {
+      firstName
+      lastName
+      scale
+      gender
+    }
+  }
+}
+`;
+
+test("Sending a query for Anakin Skywalker", async t => {
+  try {
+    const res = await sendQuery(query);
```

Comment:
I'm worried if one of those apis will go down our integration tests will stop working. 
Maybe we could have just simple hello endpoint inside the example and make call for it?
This will make things more self contained and still show use cases.